Automattic/pocket-casts-android
# Use content extension to set up fragments

## Description

This PR changes all of our fragments that manually use `ComposeView` to use `content()` extension instead. There are no functional changes. It makes things a bit cleaner and ensures that the correct `ViewCompositionStrategy` is applied.

## Testing Instructions

1. Smoke test the app and navigate through it.
2. Check things like bottom sheets and dialogs.

## Checklist
 
#### I have tested any UI changes...
